config/png/dffml_config_png/config.py

```diff
@@ -0,0 +1,37 @@
+"""
+Description of what this config does
+"""
+from typing import Dict
+from io import BytesIO
+from PIL import Image
+import numpy as np
+
+from dffml.util.entrypoint import entrypoint
+from dffml.util.cli.arg import Arg
+from dffml.base import BaseConfig
+from dffml.config.config import BaseConfigLoaderContext, BaseConfigLoader
+
+
+class PNGConfigLoaderContext(BaseConfigLoaderContext):
+    async def loadb(self, resource: bytes) -> list:
+        image_array = np.array(Image.open(BytesIO(resource)).convert("L"))
+        img_pil = Image.fromarray(image_array)
+        img_28x28 = np.array(img_pil.resize((28, 28), Image.ANTIALIAS))
+        img_784 = tuple(img_28x28.flatten())
```

Comment:
Shouldn't the scale of the image be an option rather than fixing it to 28x28. I guess this was done for MNIST, but this would be used for other images too!?

Reply:
As we can take multiple images through csv, we can also train the models without MNIST idx sources. So yeah, there's no need to keep it fixed to 28x28. I think this issue can be solved in #370 .